.github/workflows/python_lint_check.yaml

```
@@ -2,14 +2,20 @@ name: Lint check
 
 on:
   workflow_call:
+    inputs:
+      python-version:
+        description: 'List of Python versions to be used'
```

Quotes have no purpose here

```suggestion
        description: List of Python versions to be used
```